iib/web/api_v1.py

```diff
@@ -514,6 +518,14 @@ def patch_request(request_id):
             for v in value.values():
                 if not isinstance(v, list) or any(not isinstance(s, str) for s in v):
                     raise ValidationError(exc_msg)
+        elif key == 'recursive_related_bundles':
+            if not isinstance(value, list):
+                raise ValidationError(f'The value for "{key}" must be a list of non-empty strings')
```

Comment:
The check for `str` is in the very next line. I can remove the `non-empty strings` part from the msg if you feel strongly against it.  I thought it doesn't harm to give extra info in the message.

Reply:
It makes sense to have a consistent error between the two. You don't need to have non-empty there since you are not checking the content. This message would make it less likely that someone would try to fix one way then hit the other... theoretically. 

My nit would be different -- that you are raising effectively the same message in two places. Would it instead make sense to combine the checks and return the same message if either fails?

Reply:
good point. It makes sense to have a single message for both.